frontend/src/components/organisms/EventForm.tsx

```diff
@@ -143,7 +430,7 @@ const EventForm = ({ eventType }: EventFormProps) => {
               </Link>
             </div>
             <div className="col-start-1 col-span-1 sm:col-start-4 sm:col-span-1">
-              <Button type="submit" color="dark-gray">
+              <Button isLoading={isLoading} type="submit" color="dark-gray">
```

Comment:
Can you double check that this works? I couldn't see any change in the button when I click create event

Reply:
So far this is not working. We were hoping to consult you on this.